dbms/src/Storages/Page/V3/Universal/UniversalPageIdFormatImpl.h

```diff
@@ -121,17 +122,30 @@ struct UniversalPageIdFormat
         return buff.releaseStr();
     }
 
-    // LOCAL_PREFIX REGION_RAFT_PREFIX region_id RAFT_LOG_SUFFIX
+    // RAFT_PREFIX LOCAL_PREFIX REGION_RAFT_PREFIX region_id RAFT_LOG_SUFFIX
     static String toFullRaftLogPrefix(UInt64 region_id)
     {
         WriteBufferFromOwnString buff;
         writeChar(0x01, buff);
+        writeChar(0x01, buff);
```

Comment:
dose it break the existing data deployed by the older version on master?

Reply:
No, because the data is written by proxy.